ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -144,6 +149,7 @@ const Footer = () => {
       (type) => type === currentConfirmation?.type,
     );
     if (isTransactionConfirmation) {
+      ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
```

Comment:
Does the MMI build not support custom nonces?

Reply:
They are handled by the custodian 👍🏼